src/main/java/org/snomed/snowstorm/core/data/services/BranchReviewService.java

```diff
@@ -163,6 +163,11 @@ public MergeReview getMergeReviewOrThrow(String id) {
 					persistManualMergeConceptDeletion(mergeReview, conceptId);
 					logger.info("Concept {} deleted on both sides of the merge. Excluding from merge review {}.", conceptId, id);
 				} else {
+					if (sourceVersion != null && sourceVersion.isReleased() && targetVersion == null) {
+						// Deleted somewhere, whilst simultaneously versioned elsewhere.
+						mergeVersion.setAutoMergedConcept(sourceVersion);
+					}
+
```

Comment:
This block is essentially the fix. 

If the source is released and the target is null (it was deleted), then auto merge the source.